opam

```diff
@@ -16,24 +16,19 @@ tags: [
 ]
 
 build: [
-  ["ocaml" "setup.ml" "-configure" "--prefix" prefix "--%{ounit:enable}%-tests"]
-  [make "build"]
-]
-build-test: [
-  [make "test"]
-]
-install: [
-  [make "install"]
-]
-remove: [
-  ["ocamlfind" "remove" "ipaddr"]
+  [ "ocaml" "pkg/pkg.ml" "build" "--pinned" "%{pinned}%"
+    "--with-base-unix" "%{base-unix:installed}%"
+    "--with-top" "%{base-unix:installed}%" ]
 ]
```

Comment:
if you re-add a `build-test` rule, the travis CI will also compile, run, and execute those :)

```
build-test: [
  [ "ocaml" "pkg/pkg.ml" "build" "--pinned" "%{pinned}%" "--tests" "true" ]
  [ "ocaml" "pkg/pkg.ml" "test" ]
]
```